chain/transaction.go

```diff
@@ -305,106 +319,128 @@ func (t *Transaction) Execute(
 		// are set when this function is defined. If any of them are
 		// modified later, they will not be used here.
 		ts.Rollback(ctx, actionStart)
-		return &Result{false, utils.ErrBytes(rerr), maxUnits, maxFee}, nil
-	}
-	success, actionCUs, output, err := t.Action.Execute(ctx, r, ts, timestamp, t.Auth.Actor(), t.id)
-	if err != nil {
-		return handleRevert(err)
-	}
-	if len(output) == 0 && output != nil {
-		// Enforce object standardization (this is a VM bug and we should fail
-		// fast)
-		return handleRevert(ErrInvalidObject)
-	}
-	if !success {
-		ts.Rollback(ctx, actionStart)
+		return &Result{false, [][][]byte{{utils.ErrBytes(rerr)}}, maxUnits, maxFee}, nil
```

Comment:
We probably want to document somewhere that a revert will not return an error per action? Otherwise, I could imagine there being some expectation that an err is returned per action?

Reply:
I would actually not recommend deleting all previous outputs and overriding them with the revert error. Instead, I would keep what existed and then just include this error in the last action?